### PR TITLE
tidb-lightning: add contents about `REFERENCES`

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/17357
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-requirements.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ Based on the import mode and features enabled, the target database users should
    </tr>
 </table>
 
+> **Note:**
+>
+> If you have not assigned the `REFERENCES` privilege to the downstream database user, you can manually create the target table in the downstream database in advance to ensure a smooth data import.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-requirements.md:82]:
```suggestion
> If you have not granted the `REFERENCES` privilege to the downstream database user, you can manually create the target table in the downstream database in advance to ensure a smooth data import.
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 2]
```diff
@@ -77,6 +77,10 @@ Based on the import mode and features enabled, the target database users should
    </tr>
 </table>
 
+> **Note:**
+>
+> If you have not granted the `REFERENCES` privilege to the downstream database user, you can manually create the target table in the downstream database in advance to ensure a smooth data import.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-requirements.md:82]:
```suggestion
> If you have not granted the `REFERENCES` privilege to the downstream database user, you can manually create the target table in the downstream database in advance to ensure smooth data import.
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 3]
```diff
@@ -77,6 +77,10 @@ Based on the import mode and features enabled, the target database users should
    </tr>
 </table>
 
+> **Note:**
+>
+> If you have not granted the `REFERENCES` privilege to the downstream database user, you can manually create the target table in the downstream database in advance to ensure smooth data import.
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-requirements.md:82]:
```suggestion
> 
>- If the target table does not use foreign keys, the `REFERENCES` privilege is not required.
> - If you have not granted the `REFERENCES` privilege to the downstream database user, you can manually create the target table in the downstream database in advance to ensure smooth data import.
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 4]
```diff
@@ -77,6 +77,12 @@ Based on the import mode and features enabled, the target database users should
    </tr>
 </table>
 
+> **Note:**
+>
+> 
+> - If the target table does not use foreign keys, the `REFERENCES` privilege is not required.
+> - If you have not granted the `REFERENCES` privilege to the downstream database user, you can manually create the target table in the downstream database in advance to ensure smooth data import.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-requirements.md:84]:
```suggestion
> **Note:**
>
> - If the target table does not use the foreign key, the `REFERENCES` privilege is not required.
> - If the target table uses the foreign key, you need to grant the `REFERENCES` privilege for the downstream database user, or manually create the target table in the downstream database in advance to ensure smooth data import.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**